org-ql.el

```diff
@@ -27,6 +27,10 @@
 
 ;;;; Variables
 
+(if (fboundp 'org-timestamp--to-internal-time)
```

Comment:
Please put this in a new level-2 (i.e. prefixed with `;;;;`) section titled `Compatibility`, just before the `Variables` section.